packages/output/src/page.js

```diff
@@ -21,14 +21,16 @@ import PropTypes from 'prop-types';
 import { generatePatternStyles } from '@googleforcreators/patterns';
 import { PAGE_HEIGHT, PAGE_WIDTH } from '@googleforcreators/units';
 import { StoryAnimation } from '@googleforcreators/animation';
-import { ELEMENT_TYPES } from '@googleforcreators/elements';
+import {
+  ELEMENT_TYPES,
+  StoryPropTypes,
+  isElementBelowLimit,
```

Comment:
A bit unfortunate that this util has to live here, as it is more related to the editor canvas, but I guess we'll have to live with that for now. Maybe in the future we can get rid of it, as indicated by that TODO further down in this file.